### PR TITLE
[JENKINS-8927] - Document the API for creating and copying a view.

### PR DESCRIPTION
**Pull request by duemir**
See [JENKINS-8927](https://issues.jenkins.io/browse/JENKINS-8927).

The API to create a View has existed for a while but was not documented, so the above ticket was kept open. This change uses the Create Job doc as an example to add documentation about creation of the views

### Testing done

Open the `jenkins-location/api/`. The paragraphs, as on the screenshot below, should be right after the "create job" documentation.

2023-03-07-create-view-doc

No tests, since it is a documentation change

### Proposed changelog entries

- Document API for creating Views

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [ ] ~New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.~
- [ ] ~New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.~
- [ ] ~New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).~
- [ ] ~For dependency updates, there are links to external changelogs and, if possible, full differentials.~
- [ ] ~For new APIs and extension points, there is a link to at least one consumer.~

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7692"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/ModifiableViewGroup/_api.jelly
Comment (timja) [inline core/src/main/resources/hudson/model/ModifiableViewGroup/_api.jelly:6]: avoid using here or other similar concepts in links, see https://uxmovement.com/content/why-your-links-should-never-say-click-here/

possibly should also have `<code>` wrapped around it
```suggestion
        To create a new view, post <code>config.xml</code> to <a href="../createView">createView</a> with
```

File: core/src/main/resources/hudson/model/ModifiableViewGroup/_api.jelly
Comment (timja) [inline core/src/main/resources/hudson/model/ModifiableViewGroup/_api.jelly:16]:
```suggestion
        To copy a view, send a POST request to <a href="../createView">createView</a> with
```

**Review by timja — COMMENTED** [review]
Looks good, two minor comments

I haven't reviewed it rendered.

**Comment by duemir** [comment]
@timja I have committed the suggestions.
It is worth noting that `/hudson/model/ModifiableItemGroup/_api.jelly` has the same issue, i.e. `this URL`.

**Review by timja — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Pull request merged**
